ts_src/psetv2/updater.ts

```diff
@@ -373,7 +373,7 @@ export class Updater {
     if (!input.getUtxo()) {
       throw new Error('Missing input witness utxo');
     }
-    if (input.sighashType === undefined) {
+    if (!input.sighashType) {
```

Comment:
this was intentional. `!input.sighashType` returns true if sighashType is `0` which is not something we want. Can you revert?